### PR TITLE
TRUNK-4930 AdministrationService.getPresentationLocales() should return a List<Locale>

### PR DESCRIPTION
**Pull request by jtatia**
TRUNK-4930 AdministrationService.getPresentationLocales() should return a List<Locale>

I removed the TODO line at Line 297 from AdministrationService.java. Please review the work and let me know if any further changes need to be made. Thanks

### CODE REVIEW

File: api/src/main/java/org/openmrs/api/AdministrationService.java [hunk 1]
```diff
@@ -293,8 +293,6 @@
 	 * (presentation layer). This set includes all the available locales (as indicated by the
 	 * MessageSourceService) filtered by the allowed locales (as indicated by this
 	 * AdministrationService).
-	 *
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/api/AdministrationService.java]: You could leave the above empty line for formatting purposes.

File: api/src/main/java/org/openmrs/api/AdministrationService.java [hunk 2]
```diff
@@ -293,8 +293,6 @@
 	 * (presentation layer). This set includes all the available locales (as indicated by the
 	 * MessageSourceService) filtered by the allowed locales (as indicated by this
 	 * AdministrationService).
-	 * 
-	 * @return list of allowed presentation locales TODO change this return type to list?
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/api/AdministrationService.java]: We still need the return javadoc. The only test to be removed is: "TODO change this return type to list?"

Comment (jtatia) [inline api/src/main/java/org/openmrs/api/AdministrationService.java]: ok.

File: api/src/main/java/org/openmrs/api/AdministrationService.java (hunk 1 above)
Comment (jtatia) [inline api/src/main/java/org/openmrs/api/AdministrationService.java]: thanks. ill make the changes.

**Comment by dkayiwa** [comment]
@jtatia did you read this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by jtatia** [comment]
thanks for the link @dkayiwa .  Should i close the previous pull request and make a new one?

**Comment by dkayiwa** [comment]
@jtatia did you go through that link and do the needful? :)

**Comment by jtatia** [comment]
PR closed. 
checkout https://github.com/openmrs/openmrs-core/pull/1895

**Pull request closed without merging**